newspaper/network.py

```diff
@@ -57,12 +57,12 @@ def get_html_2XX_only(url, config=None, response=None):
     headers = config.headers
 
     if response is not None:
-        return _get_html_from_response(response)
+        return _get_html_from_response(response, config)
 
     response = requests.get(
```

Comment:
Upon second thought, this is the line of code you should be worried about for the PDF timeouts, not the `_get_html_from_response()` function because at that point in code the long timeout for the PDF resource has already happened.

We should move the content type check logic before this line